src/fileoverview_comment_transformer.ts

```diff
@@ -80,6 +80,10 @@ export function transformFileoverviewCommentFactory(diagnostics: ts.Diagnostic[]
     }
 
     return (sourceFile: ts.SourceFile) => {
+      if (sourceFile.fileName.endsWith('.json')) {
```

Comment:
Do you understand why only this transformer needs changes?  We register a bunch here:
https://github.com/angular/tsickle/blob/master/src/tsickle.ts#L103

Reply:
I think the other transforms do not have an effect on object literals, however this one was adding a comment at top of file which ended up causing an invalid json. 

Not sure what do you think, if we should disable all transformers for json or just this one.

Reply:
I think we should disable all transformers for json.

Reply:
So pretty much add this logic to all transformers?

Reply:
Or maybe in the place where the transformers are registered?  I'm not sure how this all fits together, you tell me :)

Reply:
I see.  I think what you had at first (just special-casing it in the one transfomer that breaks things) is fine then.